src/model.hpp

```diff
@@ -159,7 +159,7 @@ class model {
             // events that must be delivered at the start of the next
             // integration period at the latest.
             auto exchange = [&] () {
-                PE("stepping", "communciation");
+                PE("stepping", "communication");
```

Comment:
Arrrgh -- must now fix my scripts! I hadn't fixed it for that reason.
I guess it's small enough to be part of this patch set.